multicluster/controllers/multicluster/member_clusterset_controller.go

```diff
@@ -44,18 +43,26 @@ type RemoteCommonAreaGetter interface {
 	GetRemoteCommonAreaAndLocalID() (commonarea.RemoteCommonArea, string, error)
 }
 
+type leaderClusterInfo struct {
+	clusterID  string
+	serverUrl  string
+	secretName string
+}
+
 // MemberClusterSetReconciler reconciles a ClusterSet object in the member cluster deployment.
 type MemberClusterSetReconciler struct {
 	client.Client
 	Scheme    *runtime.Scheme
 	Namespace string
-	mutex     sync.RWMutex
+	// commonAreaLock protects the access to RemoteCommonArea which presents the leader cluster.
```

Comment:
Remove "which presents the leader cluster"

Reply:
Done